### PR TITLE
Feat/training add predefined back

### PR DESCRIPTION
**Pull request by christophe-rannou**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-7739
| License          | BSD 3-Clause

## Description

Add set of predefined images for training job submission

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html [hunk 1]
```diff
@@ -24,17 +24,90 @@ <h2>
...
+                                <a
+                                    target="_blank"
+                                    class="oui-button oui-button_ghost"
+                                    rel="noopener"
+                                    data-ng-href="{{:: preset.link }}"
+                                >
+                                    <span
+                                        data-translate="pci_projects_project_training_jobs_submit_select_image_preset_info"
+                                    ></span>
+                                    <i
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:56]:
```suggestion
                                    <span
```

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html [hunk 2]
```diff
@@ -24,17 +24,90 @@ <h2>
...
+                            <oui-select-picker-section
+                                class="oui-select-picker__section"
+                            >
+                                <a
+                                    target="_blank"
+                                    class="oui-button oui-button_ghost"
+                                    rel="noopener"
+                                    data-ng-href="{{ $ctrl.communityUrl }}"
+                                >
+                                    <span>OVH AI Community</span>
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:85]: No translation ?

Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:85]: No translation

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html (hunk 1 above)
Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:56]: updated

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html (hunk 2 above)
Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html]: Changed the link description to "OVHcloud Discord", no translation

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html [hunk 3]
```diff
@@ -24,17 +24,90 @@ <h2>
...
+                                class="oui-select-picker__section"
+                            >
+                                <a
+                                    target="_blank"
+                                    class="oui-button oui-button_ghost"
+                                    rel="noopener"
+                                    data-ng-href="{{ $ctrl.discordUrl }}"
+                                >
+                                    <span>OVHcloud Discord</span>
+                                    <i
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:86]: Same comment than previous one:
```suggestion
                                    <span
```

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html [hunk 4]
```diff
@@ -24,17 +24,90 @@ <h2>
...
+                                    ></span>
+                                </a>
+                            </oui-select-picker-section>
+                        </oui-select-picker>
+                    </div>
+                    <div class="col-md-6 col-xl-4 my-3">
+                        <oui-select-picker
+                            data-name="missing"
+                            data-description="{{:: 'pci_projects_project_training_jobs_submit_select_image_preset_request' | translate }}"
+                            data-picture=""
```
Comment (radireddy) [inline packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:68]:
```suggestion

```

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html [hunk 5]
```diff
@@ -24,17 +24,90 @@ <h2>
...
+                            <oui-select-picker-section
+                                class="oui-select-picker__section"
+                            >
+                                <a
+                                    target="_blank"
+                                    class="oui-button oui-button_ghost"
+                                    rel="noopener"
+                                    data-ng-href="{{ $ctrl.discordUrl }}"
+                                >
+                                    <span>OVHcloud Discord</span>
```
Comment (radireddy) [inline packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:85]: translation not required?

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html (hunk 3 above)
Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:86]: done

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html (hunk 4 above)
Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:68]: removed

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html (hunk 5 above)
Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:85]: Already discussed above, no translation  needed

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html [hunk 6]
```diff
@@ -24,17 +24,90 @@ <h2>
...
-                    'pci_projects_project_training_jobs_submit_select_image_info'
-                    | translate }}</span
-                >
-            </oui-message>
+                    data-translate="pci_projects_project_training_jobs_submit_select_image_advanced"
+                ></span>
+            </button>
             <oui-field
                 data-size="xl"
-                data-label="{{:: 'pci_projects_project_training_jobs_submit_select_image_name' | translate }}"
```
[the remark this answers is not in the document]
Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html:36]: No there's not, added it back

**Review by radireddy — APPROVED** [review]

**Pull request merged**